### ISSUE TITLE
Perhaps put "Results" higher up, or something like that

### ISSUE THREAD

**Issue by dan-tripp** [mgifford/contrast-plus#7]
Howdy.  I like this tool.  I've found a few other tools that check both APCA and WCAG 2, both most of them don't let me do both at the same time.  Yours does.  I like that.   Now here's my problem - I want to tweak my everyday workflow, when I'm wearing my auditor hat, to this: when a colour pair fails WCAG 2, and I want to know if it fails APCA too, and if so: recommend to the client a colour pair that passes both WCAG 2 and APCA.  Your tool affords that, but I often need to scroll down (from "Inputs" to "Results").  This adds about two seconds of time and effort, which adds up.  Your thoughts?

**Comment by julezrulez** [comment]
Exactly my first thought.
